### PR TITLE
Feature/storybook enhancements

### PR DESCRIPTION
**Pull request by gavinorland**
Right, here's my solution for https://github.com/UKHomeOffice/govuk-react/issues/317 to:

- Get story categories to be listed alphabetically
- Get a custom _Welcome_ story at the top
- Get this to render the `readme` file!

See what you guys think and please feel free to suggest - or commit - if you can see any better way of doing this :)

### CODE REVIEW

File: packages/storybook/.storybook/addons.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 import '@storybook/addon-actions/register';
 import '@storybook/addon-links/register';
 import '@storybook/addon-knobs/register'
+import '@storybook/addon-options/register';
```
Comment (stevesims) [inline packages/storybook/.storybook/addons.js]: i think our rules say that there should be a newline here

File: packages/storybook/.storybook/config.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
 import { configure } from '@storybook/react';
 import 'react-chromatic/storybook-addon';
+import { setOptions } from '@storybook/addon-options';
+
+setOptions({
+  name: 'govuk-react',
+  sortStoriesByKind: true,
```
Comment (stevesims) [inline packages/storybook/.storybook/config.js]: noice :+1:

File: packages/storybook/src/stories/index.js [hunk 3]
```diff
@@ -1,5 +1,11 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
 import 'normalize.css';
 import './styles.css';
+import readme from '../../../../README.md';
+
+// eslint-disable-next-line react/no-danger
+storiesOf(' Welcome', module).add('to govuk-react', () => <div style={{ padding: '10px' }} className="markdown-body" dangerouslySetInnerHTML={{ __html: readme }} />);
```
Comment (stevesims) [inline packages/storybook/src/stories/index.js]: is this not just setting raw markdown as the HTML content?

in other words, it feels like this won't actually be rendering the markdown content

we can probably use `react-markdown` to ensure that the MD gets rendered as proper HTML
that would remove the need for the `react/no-danger` lint disable

Comment (gavinorland) [reply packages/storybook/src/stories/index.js]: Yes - I had a look at using `react-markdown` but the funny thing is when the file is displayed it is _not_ markdown but is already rendered as HTML! However, the HTML is just rendered into the page unless we use `dangerouslySetInnerHTML`.

Fire it up and take a look...

**Pull request merged**